Decred Wallet/Extensions/String.swift

```diff
@@ -29,3 +29,18 @@ extension String {
         return prefix(1).uppercased() + self.lowercased().dropFirst()
     }
 }
+
+protocol Localizable {
```

Comment:
Are these protocols necessary?

Reply:
Yes to get the localized enum value of the menu title